docs/src/dedicated-gen-3/architecture/scalability.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
:clipboard: After #2907, this page can be deleted in place of https://docs.platform.sh/environments/scalability.html